src/mapd-con-es6.js

```diff
@@ -618,6 +618,294 @@ class MapdCon {
     )
   }
 
+  getUsersAsync = () =>
```

Comment:
The difference between these async functions is a single word. Maybe this could be simplified by extracting the logic to, for example, a `wrapInPromise` method?